docs/howtos/redisgraph/using-dotnet/index-usingdotnet.md

```diff
@@ -0,0 +1,114 @@
+---
+id: index-using-dotnet
+title: How to query Graph data in Redis using .NET
+sidebar_label: RedisGraph and .NET
+slug: /howtos/redisgraph/using-dotnet
+authors: [steve]
+---
+
+RedisGraph enables you to store and query graph data in Redis using the [Cypher Query Language](https://opencypher.org/). In this article, we will discuss the usage of RedisGraph with .NET.
+
+## NRedisGraph
+
+We'll use the [NRedisGraph](https://www.nuget.org/packages/NRedisGraph/) package in this tutorial. To install the package in your project, use `dotnet add package NRedisGraph`.
+
+## Running RedisGraph
+
+The easiest way to get up and running with RedisGraph locally is to use the RedisGraph docker image:
+
+```bash
+docker run -p 6379:6379 redislabs/redisgraph
+```
+
+The above command will start an instance of Redis locally with the RedisGraph module loaded, and you will be able to connect to it on **localhost:6379**
+
+## Connecting
+
+NRedisGraph makes use of the [StackExchange.Redis](https://github.com/StackExchange/StackExchange.Redis) project which is installed along with `NRedisGraph` to create the `RedisGraph` object you'll first create a `ConnectionMultiplexer`, and pull a reference, pull a reference to an `IDatabase` object from it, and then initialize the `RedisGraph` with the `IDatabase` object:
+
+```csharp
+var muxer = ConnectionMultiplexer.Connect("localhost");
+var db = muxer.GetDatabase();
+var graph = new RedisGraph(db);
+```
+
+## Sending a Query
+
+Querying in Redis Graph applies to a wide array of operations, but fundamentally when executing queries with NRedisGraph, all you need to do is execute `graph.Query` or `graph.QueryAsync` passing in the name of the graph you want to query and the query you want to run.
+
+### Creating a Node
+
+To create a node in RedisGraph, you'll use the Create Operation. Let's start by making 2 Humans, Alice and Bob:
```

Comment:
Might be good to talk about the graph being stored at a Redis key `pets` before showing code that uses that key?

Reply:
Added some text above about how pets is the key to use for the graph.